proof/crefine/RISCV64/Ipc_C.thy

```diff
@@ -312,6 +319,17 @@ lemma syscallMessage_ccorres:
               | drule nat_less_cases' | erule disjE)+
   done
 
+lemma timeoutMessage_ccorres:
```

Comment:
the `syscallMessage_ccorres` already exists on master with a silly name, given it's not a ccorres lemma but related to from_H only. given that we're not likely to rename it, I'm still unsure if consistency with it for a new lemma is better than giving it a proper name. @lsf37 ?

Reply:
I think the name is correct. It doesn’t need to be a corres predicate, it just needs to be a correspondence between Haskell and C, which it is.

Reply:
I do agree that with ccorres it does look more out of place than it does in Refine, but I think the principle is still there.